package.json

```diff
@@ -13,6 +13,7 @@
     "ms-rest": "^2.3.3",
     "ms-rest-azure": "^2.5.5",
     "rhea": "^0.2.12",
+    "azure-storage": "^2.8.2",
```

Comment:
>"azure-storage": "^2.8.2", [](start = 4, length = 26)

nothing personal, but this is a big dependency to take on - in hindsight I regret doing it in the device SDK. 
have you considered making EPH a separate module? it looks like it could fit completely separately from the rest of the SDK. Also I know EPH has its detractors so not everybody is going to want to use it, or carry the weight of the storage SDK.
On top of that, when you introduce dependency scanning, any vulnerability in the dependencies of the storage SDK will apply to the whole event hubs SDK instead of just the EPH part of it

Reply:
Yup. Agree with that. I can make this a separate package with the name`azure-event-processor-host`

Reply:
Will have the eph code in this repo. Will have two separate top level folders (client and eph).
Once the review is complete, will get this PR merged as-is. Will send another PR specifically to reorganize the repo and have separate packages.